components/apimgt/org.wso2.carbon.apimgt.rest.api.core/src/gen/java/org/wso2/carbon/apimgt/rest/api/core/dto/APIInfoDTO.java

```diff
@@ -0,0 +1,166 @@
+package org.wso2.carbon.apimgt.rest.api.core.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.Objects;
+
+/**
+ * APIInfoDTO
```

Comment:
This is an auto generated class when creating a new REST API. So, normally we do not change anything in these classes.